Adyen/UI/List/CoreDataSource.swift

```diff
@@ -0,0 +1,127 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+internal protocol ListViewControllerDataSource: NSObject, UITableViewDataSource {
+    
+    var sections: [ListSection] { get }
+    
+    var cellReuseIdentifier: String { get }
+    
+    func reload(newSections: [ListSection], tableView: UITableView)
+    
+    func deleteItem(at indexPath: IndexPath, tableView: UITableView)
+    
+    func cell(for tableView: UITableView, at indexPath: IndexPath) -> UITableViewCell
+    
+    func startLoading(for item: ListItem, _ tableView: UITableView)
+    
+    func stopLoading(_ tableView: UITableView)
+}
+
+internal final class CoreDataSource: NSObject, ListViewControllerDataSource {
+    
+    internal var sections: [ListSection] = []
+    
+    internal let cellReuseIdentifier = "Cell"
```

Comment:
I guess it is ok for now, but in future would be nice to support multiple identifiers for "Instant payment method" cells.